### PR TITLE
New custom var dialog

### PR DESCRIPTION
**Pull request by eumario**
The new Custom Variable Dialog designed using VisUI.  Should allow for easier reading on different screen resolutions, as the list of variables are no longer boxed to small dimensions on lower resolutions.

### CODE REVIEW

**Comment by azakhary** [comment]
Awesome, let me pull this to take a look! 
Btw, while I am on it could you please commit the Apache License part back, probably your IDE deleted it.

**Comment by eumario** [comment]
Not a problem, adding it back in here, give me a moment. lol

**Comment by eumario** [comment]
Header added back in, and garbage removed from old version of the Custom Variable Dialog.

**Comment by azakhary** [comment]
Looks great, one thing, when dialog is opened and I click custom vars dialog, it opens second one, instead nothing should happen.

**Comment by azakhary** [comment]
Also, when you set a variable to an item (or do any change to it) you need to make sure it's VO object is updated as well, otherwise if you save project, and then open it again, data will be lost, because data is saved from VO. in order to fix that, after modifying anything on an item, you need to call item.updateDataVO();

File: src/com/uwsoft/editor/gdx/ui/dialogs/CustomVariablesDialog.java [hunk 1]
```diff
@@ -20,179 +20,185 @@
...
+		
+		add(pane).expandX().expandY().fillX().fillY();
+		add(new Separator()).padTop(10).fillY().expandY();
+		add(form).row();
+		
+		add(newKVE).colspan(3).expandX().fillX().padBottom(3).padTop(3).row();
+	}
+	
+	private void setupHooks() {
+		varNames.addListener(new ChangeListener() {
```
Comment (azakhary) [inline src/com/uwsoft/editor/gdx/ui/dialogs/CustomVariablesDialog.java]: After changing the variable, you need to "update" the item, by calling item.updateDataVO();

**Comment by eumario** [comment]
Latest commit adds updates.

**Comment by azakhary** [comment]
Thank you! I am going to merge this

**Pull request merged**